app/controllers/casework/decisions_controller.rb

```diff
@@ -2,7 +2,11 @@ module Casework
   class DecisionsController < Casework::BaseController
     include SetDecisionAndAuthorise
 
-    before_action :set_decision, except: [:create]
+    before_action :set_decision, except: [:create, :index]
+
```

Comment:
👇 completes#show is replaced by this list of decisions. At the bottom of the list acaseworker can either: send decision (complete), add another, or, return to their list.

<img width="1120" alt="Screenshot 2024-10-24 at 07 01 48" src="https://github.com/user-attachments/assets/d1d094bf-4b7c-4662-b6cb-aa5d79cffe63">